packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -62,6 +65,49 @@ class TransloaditAssembly extends Emitter {
     this.close()
   }
 
+  #connectServerEvent () {
+    this.#sse = new EventSource(this.status.assembly_ssl_url)
+
+    this.#sse.addEventListener('open', () => {
+      if (this.socket) {
```

Comment:
why do we disconnect websocket here?

Reply:
If we have SSE working, we don't need a websocket (websockets are more expensive, and deliver the exact same info).